app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
```

Comment:
It's a good practice to add a blank line between global imports and local imports in import statements for better code readability and organization